.github/CONTRIBUTING.md

```diff
@@ -44,10 +44,8 @@ To make review of PRs easier, please:
 
 ## Commit and PR Messages
 
-* **Reference issues, wiki pages, and pull requests liberally!**
+* Ensure you follow the [angular style](https://github.com/angular/angular.js/blob/master/DEVELOPERS.md) for commit messages. See [here](https://github.com/conventional-changelog-archived-repos/conventional-changelog-angular/blob/master/convention.md) for an outline of the structure used. **This is enforced for at least one commit in every PR to make the automatic semantic versioning work.**
```

Comment:
i think we should explain what that is here, then link.

don't just link. we're not in charge of what their guidelines look like